### DISCUSSION TITLE
[question] 请问tiny-engine里面的几个基础概念如何简单理解。

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#871] [Q&A] [answered]
请问tiny-engine里面的几个基础概念如何简单理解：
- 物料
- 插件
- 平台
- 组件（组件是否就是物料呢？）
- 区块
- schema

**Comment by chilingling** [comment, marked as answer]
您可以看下官方文档：https://opentiny.design/tiny-engine#/help-center/course/engine
